test/runtests.jl

```diff
@@ -0,0 +1,92 @@
+using CacheVariables, BSON, Test
+
+## Add data directory, define data file path
+dirpath = joinpath(@__DIR__, "data")
+!isdir(dirpath) && mkdir(dirpath)
```

Comment:
```suggestion
isdir(dirpath) && rm(dirpath; recursive=true)
mkdir(dirpath)
```
Just in case a previous run did not finish cleaning up.